src/Exception/DisposableEmailException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PCF\DisposableEmail\Exception;
+
+class DisposableEmailException extends \Exception
+{}
```

Comment:
Opening class brace must be on a line by itself
Closing class brace must be on a line by itself